src/schemathesis/loaders.py

```diff
@@ -126,6 +133,7 @@ def from_dict(
             method=method,
             endpoint=endpoint,
             tag=tag,
+            operationId=None,  # Swagger does not support temporarily
```

Comment:
How Swagger is different in this matter? How much effort do we need to support it?